src/styles/bs5/summernote-bs5.js

```diff
@@ -175,8 +175,8 @@ const ui = function(editorOptions) {
 
     button: function($node, options) {
       return renderer.create('<button type="button" class="note-btn btn btn-outline-secondary btn-sm" tabindex="-1">', function($node, options) {
-        if (options && options.data && options.data.toggle === 'dropdown') {
-          $node.removeAttr('data-toggle');
+        if (options && options.data && options.data['bs-toggle'] === 'dropdown') {
```

Comment:
I think this change is wrong. It needs to be
`        if (options && options.data && options.data['toggle'] === 'dropdown') {`
otherwise it is still not working for me.

Reply:
It needs to be `if (options && options.data && options.data['toggle'] === 'dropdown') {`